one_fm/fixtures/property_setter.json

```diff
@@ -1,20 +1,4 @@
 [
- {
```

Comment:
You removed a lot of fields from here, was it intentional ?

Reply:
Yes. Refactoring as per the new process.